src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -491,14 +457,18 @@ const updatedTodo = await API.graphql({
 
 </BlockSwitcher>
 
-Notes:
+<Callout>
+
+__Notes:__
+
+- You do not have to pass in the `updatedAt` field. AWS AppSync manages this for you.
+- If you pass in *extra* input fields not expected by the AWS AppSync schema, this query will fail. You can see this in the `error` field returned by the query. In GraphQL, errors are not thrown like exceptions in other languages. Instead, any errors are captured and returned as part of the query result in the `error` field.
 
-- You do not have to pass in `createdAt` and `updatedAt` fields. AppSync manages this for you.
-- If you pass in *extra* input fields not expected by the AppSync schema, this query will fail. You can see this in the `error` field returned by the query. In GraphQL, errors are not thrown like exceptions in other languages. Instead, any errors are captured and returned as part of the query result in the `error` field.
+</Callout>
 
-#### Deleting an item
+#### Delete an item
 
-You can then delete the todo by using the delete mutation:
+You can then delete the Todo by using the delete mutation. Only an `id` is needed to identify which item to delete:
```

Comment:
I moved up the sentence after the code block about the id b/c it seemed out of place at the bottom - does it need to move back?